modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -547,6 +547,33 @@ Use the Stats API to view your entire organization's usage of Object Store for b
 * Usage per region
 * Usage per store
 
+
+[[osv2-usage-limits]]
+=== Usage Metrics Limits
+
+[%header,cols="10,15,15a"]
+|===
+| Parameter | Description | Values
+| Time series (`period`) | Interval that the query records |
+
+* `1hour`
+* `1day`
+* `1month`	
+
+|Data retention | Length of time that Anypoint MQ retains the data, based on `period`| 
+
+* `period=1hour`: 3 months 
+* `period=1day`: 12 months
+* `period=1month`: 24 months
+
+| Query timeframe | Maximum interval between `startDate` and `endDate` in the query, based on `period` |
+
+* `period=1hour`: 1 month 
+* `period=1day`: 6 months
```

Comment:
@hannanelson I discussed with metering team and there is an update here. We will be providing this higher time frames only for some allowlisted customers who are currently already using these timeframes (this is to minimize customer impact for now). But all other customers will be allowed smaller timeframes and that is what we want to document. I updated my doc accordingly: https://salesforce.quip.com/RtutAdi5RsiH#temp:C:DDD6ae61dfd9dda462cb9985b755